CHANGES.md

```diff
@@ -48,21 +69,19 @@ To be released.
 
  -  The following store URI schemes are deprecated:  [[#1573], [#1955]]
 
-     - `default`: Use `default+file` instead.
-     - `rocksdb`: Use `rocksdb+file` instead.
-
 [#1359]: https://github.com/planetarium/libplanet/issues/1359
 [#1573]: https://github.com/planetarium/libplanet/issues/1573
 [#1953]: https://github.com/planetarium/libplanet/issues/1953
 [#1955]: https://github.com/planetarium/libplanet/pull/1955
+[#1957]: https://github.com/planetarium/libplanet/issues/1957
 [#1959]: https://github.com/planetarium/libplanet/issues/1959
 [#1960]: https://github.com/planetarium/libplanet/pull/1960
 [#1961]: https://github.com/planetarium/libplanet/pull/1961
+[#1962]: https://github.com/planetarium/libplanet/pull/1962
 [#1966]: https://github.com/planetarium/libplanet/pull/1966
 
 
 Version 0.34.0
---------------
```

Comment:
You probably made some mistakes during rebasing.

Reply:
Should be all fixed I think...? Can you check again for me? 😅